.github/workflows/main.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     needs: [formatting]
     strategy:
       matrix:
-        java: [8, 11, 17]
+        java: [17,19]
```

Comment:
Jakarta EE 10 requires Java SE 11 and supports 17. We need to include that. Also, does Payara 6 really officially support 19 yet?

Reply:
add Java 11